apps/mull-ui/src/app/pages/messages/event-chat/event-chat.tsx

```diff
@@ -25,18 +25,19 @@ interface subscriptionData {
 }
 
 export interface EventChatProps {
-  eventId: number;
   channelName: string;
   restrictChatInput: boolean;
 }
 
-export const EventChat = ({ eventId, channelName, restrictChatInput }: EventChatProps) => {
+export const EventChat = ({ channelName, restrictChatInput }: EventChatProps) => {
```

Comment:
Missing the [back button ](https://www.figma.com/file/FJPXMYI4ULCnuCEZvPs67k/Mull?node-id=1749%3A28348)that was added after the Messages Main Menu redesign